friends/src/components/Friend.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { update, deleteFriend } from '../actions';
+
+class Friend extends Component {
+    constructor() {
+        super();
+        this.state = {
+            editingId: null,
```

Comment:
Hmm ... questionable use of component state. If editingId always equals id ... why have both?